app/post_results.go

```diff
@@ -74,6 +74,12 @@ type tlogEntry struct {
 	IntegratedTime int64  `json:"integratedTime"`
 }
 
+//go:embed fulcio_v1.crt.pem
+var fulcioRoot []byte
+
+//go:embed fulcio__intermediate.crt.pem
```

Comment:
Typo - double underscore ?

Reply:
interesting... i think that fixed the typecheck too!